src/components/record/related-table-actions.tsx

```diff
@@ -445,48 +532,68 @@ const RelatedTableActions = ({
     }
 
     return (
-      <ChaiseTooltip
-        placement='top'
-        tooltip={tooltip}
-      >
-        <button className='chaise-btn chaise-btn-secondary toggle-display-link' onClick={onToggleDisplayMode}>
+      <ChaiseTooltip placement='top' tooltip={tooltip}>
+        <button
+          className={`chaise-btn toggle-display-link ${
+            tertiary ? 'chaise-btn-tertiary dropdown-button' : 'chaise-btn-secondary'
+          }`}
+          onClick={tertiary ? undefined : onToggleDisplayMode}
+        >
           <span className={`chaise-btn-icon ${icon}`}></span>
           <span>{label}</span>
         </button>
       </ChaiseTooltip>
-    )
+    );
   };
 
   const renderCreateBtnTooltip = () => {
     if (relatedModel.canCreateDisabled) {
       let fkr;
       if (relatedModel.initialReference.derivedAssociationReference) {
-        fkr = relatedModel.initialReference.derivedAssociationReference.origFKR
+        fkr = relatedModel.initialReference.derivedAssociationReference.origFKR;
       } else {
         fkr = relatedModel.initialReference.origFKR;
       }
-      const keyset = (<code>{fkr.key.colset.columns.map((c: any) => c.name).join(', ')}</code>);
+      const keyset = <code>{fkr.key.colset.columns.map((c: any) => c.name).join(', ')}</code>;
 
       if (relatedModel.isPureBinary) {
-        return <span>Unable to connect to {currentTable} records until {keyset} in this {mainTable} is set.</span>;
+        return (
+          <span>
+            Unable to connect to {currentTable} records until {keyset} in this {mainTable} is set.
+          </span>
+        );
       }
-      return <span>Unable to create {currentTable} records for this {mainTable} until {keyset} in this {mainTable} is set.</span>
+      return (
+        <span>
+          Unable to create {currentTable} records for this {mainTable} until {keyset} in this{' '}
+          {mainTable} is set.
+        </span>
+      );
     }
 
     if (relatedModel.isPureBinary) {
-      return <span>Connect {currentTable} records to this {mainTable}.</span>;
+      return (
+        <span>
+          Connect {currentTable} records to this {mainTable}.
+        </span>
+      );
     }
-    return <span>Create {currentTable} records for this {mainTable}.</span>
+    return (
+      <span>
+        Create {currentTable} records for this {mainTable}.
+      </span>
+    );
   };
 
-  return (
-    <>
-      <div className='related-table-actions' ref={container}>
-        {relatedModel.canCreate &&
-          <ChaiseTooltip
-            placement='top'
-            tooltip={renderCreateBtnTooltip()}
-          >
+  const renderButtons = () => {
+    return (
```

Comment:
As far as I could tell, you're doing the exact same thing as `renderButton`. I see a lot of duplicated code here. Ideally there should be just one function rendering the button. So ideally this function should call your `renderButton` function.

Reply:
By doing this, we will avoid issues where we forget to update one of them. For example right now these two functions are returning slightly different results for "Bulk Edit" button.